.github/workflows/trigger_all_tests.yml

```diff
@@ -13,11 +13,13 @@ jobs:
 
```

Comment:
Maybe we can add another `if` (right above this line 😅) to make sure this doesn't work on forks? 

```yaml
if: github.repository == 'RevenueCat/purchases-ios'
```

Reply:
I think that check could also be removed in the fork though? but yes, another extra check I guess

Reply:
Yea I think so too 🤔 But I see this being done, e.g. here: https://github.com/cashapp/multiplatform-paging/blob/main-3.3.0-alpha02/.github/workflows/release.yaml#L25

Reply:
yeah... will add it since I think it shouldn't hurt in any case 👍